### DISCUSSION TITLE
bot runs and prints in vscode but doesn't run when I schedule it to run using Windows scheduler

### DISCUSSION THREAD

**Discussion by myitran97** [Rapptz/discord.py#9672] [Q&A]
I am new to this. I have a question. I wrote a bot to print a message to my server and it ran when I press shift+enter in vscode, but when i schedule using windows scheduler to run the .py script at a specific time, it doesn't run. It also return RuntimeError: asyncio.run() cannot be called from a running event loop when I ran it not using vscode too. What is the problem with it?
Is there a way to make the bot print a message with fixed timing, without having the script run in a loop? Please help, I really appreciate it

**Comment by bandwiches** [comment]
Thoughts off the top of my head...

- Are you using `virtualenv` ?
- When you call your script from windows scheduler are you sourcing from the same directory you are when in when using vs code?

Your last question needs a little more context, you can print messages based on `datetime` and loops in various ways, asyncio is not required for that.
